src/views/PrimaryTypeNode.ts

```diff
@@ -34,6 +34,10 @@ export class PrimaryTypeNode extends DataNode {
         return "";
     }
 
+    public getLabel(): string {
```

Comment:
Instead of introducing `getLabel()` into all different types. What about directly set the displayName at java side?